MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include *.py
+include requirements.txt
+include README.md
+include LICENSE
+include setup.py
```

Comment:
should we remove this since it's redundant?

Reply:
I will make the python file inclusion explicit.